scripts/setup/apt-repos/ksplice/focal.list

```diff
@@ -0,0 +1,2 @@
+deb http://www.ksplice.com/apt focal ksplice
+deb-src http://www.ksplice.com/apt focal ksplice
```

Comment:
I think this duplication is nicer than the one we used to have, but what's the thinking around having these pre-substituted files, rather than a file with a variable in it for each category?

Reply:
It seemed easiest to understand and write; we'd need to split into a `debian.list` and `ubuntu.list` because Zulip's files are different between them, and it also serves to also straightforwardly encode what possible distro releases each of them is expected to be used on.

Reply:
I think the main downside is that it increases the manual work required to support new Debian/Ubuntu releases slightly, but that's a somewhat rare operation, and still easy to find everything with e.g. `git grep focal`, so I don't think it's a problem.

I'm fine with deferring to the person working on this; we can always fix forward if we decide we want the template model after all.

@ligmitz @andersk thoughts?  Assuming no objections from you folks, I think this is ready to merge.

Reply:
Yeah, I think the process of supporting a new release does not take place very often and considering that we have multiple distros supported, adding support for a new one is easy to do so on the lines of other supported distros.